frontend-react/src/shared/CodeSnippet/CodeSnippet.tsx

```diff
@@ -0,0 +1,161 @@
+import React, { ReactNode, useState } from "react";
+import { Icon, Tooltip } from "@trussworks/react-uswds";
+import classnames from "classnames";
+
+import styles from "./CodeSnippet.module.scss";
+
+/**
+ * Gets the text from the ReactNode object.
+ */
+const getNodeText = (node: React.ReactNode): string => {
```

Comment:
Is all this code for the syntax highlighting? I highly recommend we use one of the many well-maintained packages to do this for us! It'll save us quite a lot of time / headache down the road rather than maintaining our own.

Reply:
getNodeText is a method to extract the text out of a node object. This is independent of the syntax highlighting

Reply:
I remove the createHighlightedText method